spec/requests/self_assessment_pages_spec.rb

```diff
@@ -69,7 +69,7 @@
         click_button 'Save Changes'
         current_path.should == review_self_assessment_path(review, self_assessment)
 
-        page.should have_selector('p.field-error-message', text: 'can\'t be blank')
+        page.should have_selector('div.field > p.field-error-message', text: 'can\'t be blank')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.